### DISCUSSION TITLE
PyTorch Lightning / HPC Support

### DISCUSSION THREAD

**Discussion by jomar0** [jeshraghian/snntorch#260] [Q&A] [answered]
Hi,
I was wondering if the framework supported PyTorch Lightning / DDP training on HPC Clusters as Norse (www.github.com/norse/norse) does.
If not, would you consider adding support?
Thanks

**Comment by jeshraghian** [comment, marked as answer]
snnTorch uses PyTorch as its backend, so it's fully compatible with PyTorch Lightning & DDP training in much the same way you would use a regular PyTorch model.
